app/controllers/users/request_orders_controller.rb

```diff
@@ -6,6 +6,7 @@ def index
 
     def show
       @request_order = current_business.request_orders.find_by(uuid: params[:uuid])
+      @sub_request_orders = @request_order.order.request_orders.where(parent_id: @request_order.id)
```

Comment:
以下でいけそう。cf: https://github.com/ClosureTree/closure_tree#eager-loading
```suggestion
      @sub_request_orders = @request_order.children
```

Reply:
なるほど、ありがとうございます！後程確認します！